Sources/Actions/Login2.php

```diff
@@ -24,6 +24,7 @@
 use SMF\ErrorHandler;
 use SMF\IntegrationHook;
 use SMF\Lang;
+use SMF\QueryString;
```

Comment:
```suggestion
use SMF\IP;
```

This line will also need to be moved up in order to keep PHP-CS-Fixer happy.